### PR TITLE
Update metal-roughness texture packing

### PR DESCRIPTION
**Pull request by sbtron**
1) Update packing info for metal roughness texture #857
2) Add formulas for occlusion strength #884 and normal scale #885

### CODE REVIEW

**Comment by sbtron** [comment]
Merging since changes have been been discussed in linked Issues. Leave a comment if language needs to be updated.

File: specification/2.0/schema/material.pbrMetallicRoughness.schema.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "metallicRoughnessTexture": {
             "allOf": [ { "$ref": "textureInfo.schema.json" } ],
             "description": "The metallic-roughness texture.",
-            "gltf_detailedDescription": "The metallic-roughness texture has two components. The first component (R) contains the metallic-ness of the material. The second component (G) contains the roughness of the material. These values are linear. If the third component (B) and/or the fourth component (A) are present, they are ignored."
+            "gltf_detailedDescription": "The metallic-roughness texture. The metalness values are sampled from the G channel. The roughness values are sampled from the B channel. These values are linear. If other channels are present (R or A), they are ignored for metallic-roughness calculations."
```
Comment (emackey) [inline specification/2.0/schema/material.pbrMetallicRoughness.schema.json]: `G` is roughness, `B` is metalness, per #857.

**Comment by emackey** [comment]
Other than that, looks good to me.

File: specification/2.0/schema/material.pbrMetallicRoughness.schema.json (hunk 1 above)
Comment (sbtron) [inline specification/2.0/schema/material.pbrMetallicRoughness.schema.json]: Oops yes thanks for catching this. Updating.....

Comment (sbtron) [inline specification/2.0/schema/material.pbrMetallicRoughness.schema.json]: Fixed in #897 thanks again!

**Pull request merged**